packages/office-addin-dev-settings/src/dev-settings.ts

```diff
@@ -208,6 +214,15 @@ export async function setSourceBundleUrl(addinId: string, components: SourceBund
   }
 }
 
+export async function switchWebView(webView: WebViewType) {
+  if (process.platform !== "win32") throw new Error("switchWebView only usable on Win32");
```

Comment:
This should have the platform-specific code use the same function name and parameters, so it just implements the code as required for the platform.

Reply:
This should also probably be `WebViewType | undefined` so the `undefined` value would clear the setting.

Reply:
Maybe call this `setWebViewType()`